packages/sw-runtime-caching/src/lib/stale-while-revalidate.js

```diff
@@ -24,6 +24,10 @@ import assert from '../../../../lib/assert';
  *
  * In addition to updating the appropriate caches, it will also trigger any
  * appropriate plugins defined in the underlying `RequestWrapper`.
+ * 
+ * This class always returns the cache and returns it first regardless of how
```

Comment:
> The network response is used to update the cache asynchronously, after the previously cached response is returned to fulfill the request.

So I'd have to call this again to get what returned in the validate part of staleWhileRevalidate?